bosch_locator_bridge/include/bosch_locator_bridge/locator_bridge_node.hpp

```diff
@@ -65,6 +65,9 @@ class LocatorBridgeNode : public rclcpp::Node
   bool check_module_versions(
     const std::unordered_map<std::string, std::pair<int32_t,
     int32_t>> & module_versions);
+  bool get_config_entry(const std::string & name, std::string & value) const;
```

Comment:
Might it make sense to have this templated to for completness with set_config_entry? There is a Poco::Dynamic::Var::convert method which could be used.